samples/keys/hw_unique_key/README.rst

```diff
@@ -0,0 +1,96 @@
+.. _hw_unique_key_usage:
+
+Hardware unique key
+###################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample shows how to use a hardware unique key (HUK) to derive an encryption key through psa_crypto APIs.
+
+Overview
+********
+
+The sample goes through the following steps to derive a key and use it to encrypt a string:
+
+1. Initializing the hardware, if applicable.
+
+   When TF-M is enabled, the hardware is initialized automatically when booting.
+
+#. Generating HUKs, if applicable.
+
+   When TF-M is enabled, the HUKs are generated automatically when booting.
+
+#. Generating a random initialization vector (IV).
+
+#. Deriving a key used for encryption.
+
+   When TF-M is enabled, the key is derived using the psa_crypto APIs.
+   Otherwise, the native APIs are used, and the key is imported into psa_crypto.
+
+#. Encrypting a sample string using the key_id received from the psa_crypto.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf5340dk_nrf5340_cpuapp, nrf5340dk_nrf5340_cpuappns, nrf9160dk_nrf9160, nrf9160dk_nrf9160ns, nrf52840dk_nrf52840
```

Comment:
What is nrf5340dk_nrf5340_cpuappns? Is it a typo?
I know we have nrf5340dk_nrf5340_cpuapp and nrf5340dk_nrf5340_cpunet

Reply:
It is the non-secure variant of the board, and not a typo.

Reply:
Hm... It's crashing the doc build. Let me check this. It must be a typo.

Reply:
It should be: nrf5340dk_nrf5340_cpuapp_ns

Reply:
Or, I've seen a couple of these around as well: nrf5340dk_nrf5340_cpuapp_and_cpuapp_ns

Reply:
Fixed, ns -> _ns